README.md

```diff
@@ -100,36 +96,48 @@ Check [Sentry-java](https://github.com/getsentry/sentry-java) documentation to c
 The Logback Appender is automatically configured by the plugin, you just have to set enabled environments as shown in Configuration section.
 
 All application exceptions will be logged on sentry by the appender.
-The appender is configured to log just the `ERROR` and `WARN` levels.
+The appender is configured to log just the `WARN` level and up.
 To log manually just use the `log.error()` method.
 
-## sentryClient
+## Capturing events manually
 
-You also can use `sentryClient` to sent info messages to Sentry:
+You also can use `Sentry` class to sent info messages to Sentry:
 
 ```groovy
-import io.sentry.SentryClient
-import io.sentry.event.Event
-import io.sentry.event.EventBuilder
-import io.sentry.event.interfaces.ExceptionInterface
-
-SentryClient sentryClient // To inject Spring bean raven client in your controllers or services
+import io.sentry.Sentry
+import io.sentry.SentryEvent
+import io.sentry.SentryLevel
+import io.sentry.protocol.Message
 
 // Send simple message
-sentryClient?.sendMessage("some message")
+Sentry.currentHub.captureMessage("some message")
 
 // Send exception
-sentryClient?.sendException(new Exception("some exception"))
+Sentry.currentHub.captureException(new Exception("some exception"))
 
 // Custom event
-EventBuilder eventBuilder = new EventBuilder()
-           .withMessage("This is a test")
-           .withLevel(Event.Level.INFO)
-           .withLogger(MyClass.class.name)
+SentryEvent event = new SentryEvent(
+        message: new Message(message: "This is a test"),
+        level: SentryLevel.INFO,
+        logger: MyClass.class.name,
+)
 
-sentryClient?.sendEvent(eventBuilder.build())
+Sentry.currentHub.captureEvent(event)
 ```
 
+## Performance monitoring
+
+Sentry [performance monitoring](https://docs.sentry.io/platforms/java/performance/) is disabled by default.
+You have to set config `tracesSampleRate` to value greater than `0` to enable it.
+[Sample rate](https://docs.sentry.io/platforms/java/configuration/sampling/#sampling-transaction-events) represents percentage of requests that should be traced (value `1` = 100%).
+Requests handled by controllers are then automatically traced in Sentry according to sample rate. 
+You can also enable [JDBC tracing](https://docs.sentry.io/platforms/java/performance/instrumentation/jdbc/) by setting config `traceJDBC` to `true`.
+
+To enable [distributed tracing](https://docs.sentry.io/product/sentry-basics/tracing/distributed-tracing/) you need to [connect services](https://docs.sentry.io/platforms/java/performance/connect-services/) (frontend) 
+by adding `<sentry:traceMeta />` to `<head>` of your gsp view. 
+
+If you configure `linkPrefix` you can also add `<sentry:traceLink />` to your gsp view which displays direct link to transaction in sentry application.
+
 # Latest releases
 
 * 2018-07-16 **V11.7.25** : upgrade Sentry java lib to 1.7.25 + stack trace sanitizer
```

Comment:
Latest release entry is needed ;) Possibly if @musketyr doesn't want this new version in their repo, they can at least add a link to this fresher version in their README.

Reply:
I will happily do so. I don't think we're going to adopt the version 5 for our Grails apps.

Reply:
I didn't know when and what version will be released and what all will it contain. I assumed the entry would be created just before release.